### ISSUE TITLE
Adding custom font

### ISSUE THREAD

**Issue by NadimHallaq** [reshaped/community#70]
Me again! I'm trying to add a custom font, however I don't understand what you mean by this line: **If you're using a custom font in this token, don't forget to include your font file into the product** in the docs [here](https://reshaped.so/content/docs/theming/creating-themes).  

Does the custom font go in the generated src file, or do I keep it my public/fonts and import it in reshaped.config? or neither :)

**Comment by blvdmitry** [comment]
Hey there! 
When you're creating a theme, you can pick a custom font family which will be used in CSS. In case you're applying a font that's not available natively, you'll have to make sure that a file with that font is included either in CSS or in HTML. 

For example, if you're using a font from Google Fonts in your theme, you'll need to include their font link in your HTML:
<img width="318" alt="image" src="https://user-images.githubusercontent.com/887379/174437181-1096a810-9cac-4d92-b30a-81644648d7b0.png">

**Comment by NadimHallaq** [comment]
Okay, I did not have a global css file imported in my remix root.tsx, so I made global.css in my styles folder and added a Link to the stylesheet. 

I was only relying on the generated theme.css file to add my font, but it reset every time I did ` npm run build:themes`. 

So basically if your using Remix, you'll need to add three stylesheets, theme, styles, and global.css if your adding a custom font! 

Works perfectly. Thank you.
